TAF_EasyRest/TestFramework/PageComponents/Owner/ManageMenuPageComponent.cs

```diff
@@ -0,0 +1,31 @@
+﻿using TestFramework.Pages.Owner;
+
+namespace TestFramework.PageComponents.Owner
+{
+    public class ManageMenuesPageComponent
+    {
+        private IWebDriver driver;
```

Comment:
I suppose it'd be better to add property here private "IWebDriver driver { get; }"